sherpa/plot/bokeh_backend.py

```diff
@@ -197,6 +197,24 @@ def __exit__(self, exec_type, value, traceback):
         '''Called from the UI after an interactive plot is done.'''
         return False
 
+    def get_latex_for_string(self, txt):
+        """Convert LaTeX formula
+
+        Parameters
+        ----------
+        txt : str
+            The text component in LaTeX form (e.g. r'\alpha^2'). It
+            should not contain any non-LaTeX content.
+
+        Returns
+        -------
+        latex : str
+            The text modified as appropriate for a backend so that the LaTeX
+            will be displayed properly.
+
+        """
+        return f"$${txt}$$"
+
```

Comment:
So, does this imply a minimum version of bokeh we support (or that if we have an older version we just don't get nice plot labels)?

Reply:
For an older version it's just going to look like it does in #1973 (except that you'll see a double $$, instead of a single $). So, no formal minimum version, but this will make it look better on newer version (not sure the exact number where they changed that; there is an entry in the changelog for bokeh 3.4 that could be related, but I don't really want to spend the time to read through and understand the full PR to see if that's the right one)

Reply:
Just wanted to check if we needed to add a note about bokeh version, but it seems like it's not worth it.